Lib/distutils/tests/test_upload.py

```diff
@@ -128,6 +128,32 @@ def test_upload(self):
         auth = self.last_open.req.headers['Authorization']
         self.assertNotIn('\n', auth)
 
+    # bpo-32304: archives whose last byte was b'\r' were corrupted due to
+    # normalization intended for Mac OS 9.
+    def test_upload_correct_cr(self):
```

Comment:
This is the test failure: https://travis-ci.org/python/cpython/jobs/333550360#L2640-L2647

Reply:
Yeah, I see what's going on - the [cmd.show_response](https://github.com/python/cpython/pull/5331/files/85bb01b584c2026aaf4ff471e262a08d143f58de#diff-560972711f060c4e2db13d7d9067a84bR149) isn't the same in this version. There are some other differences in the test I'll smooth out shortly.